packages/babel-plugin/src/types.tsx

```diff
@@ -65,6 +65,14 @@ export interface State extends PluginPass {
     styled?: string;
   };
 
+  /**
+   * Details of pragmas that are currently enabled in the pass.
+   */
+  pragma: {
+    jsxImportSource?: boolean;
```

Comment:
can both be enabled at the same time?

Reply:
Technically yes, but Babel or any bundler would throw